docs/concepts/services-networking/custom-dns.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
Travis is failing and points to this YAML as the reason: https://travis-ci.org/kubernetes/website/builds/310360212?utm_source=github_status&utm_medium=notification

The code validates correctly (and even lints well!), so I'm not sure what's going on. The `{% include %}` statement that invokes it likewise appears correct. @chenopis, any insight?

Reply:
@chenopis The Travis build: https://travis-ci.org/kubernetes/website/builds/310360212?utm_source=github_status&utm_medium=notification

Reply:
fixed